xdelta3-dir-patcher

```diff
@@ -65,55 +449,81 @@ class XDelta3DirPatcher(object):
 
     # TODO: Unit test me
     def copy_attributes(self, src_file, dest_file):
-        if args.debug: print("Copying mode data")
+        if self.args.verbose: print("Copying file metadata:", dest_file)
         copymode(src_file, dest_file)
-
-        if args.debug: print("Copying stat data (sans UID/GID")
         copystat(src_file, dest_file)
 
-        if args.debug: print("Copying UID & GID")
         uid = stat(src_file).st_uid
         gid = stat(src_file).st_gid
 
         chown(dest_file,uid,gid)
 
-    def _find_file_delta(self, rel_path, new_file, old_root, new_root, target_root):
-        print("Processing %s" % new_file)
+    def _find_file_delta(self, filename, old_archive_obj, new_archive_obj,
```

Comment:
I think this should probably be called `_create_file_delta` since it's not just figuring out the delta but creating the diff in the target, too.